src/VisualStudio/Core/Impl/SolutionExplorer/AnalyzerItem/AnalyzerItem.cs

```diff
@@ -9,83 +9,73 @@
 using Microsoft.VisualStudio.Imaging;
 using Microsoft.VisualStudio.Imaging.Interop;
 
-namespace Microsoft.VisualStudio.LanguageServices.Implementation.SolutionExplorer
```

Comment:
siwtch to file scpoed namespace, primary constructor and normal autoprops.

Reply:
most of this PR is just clenaup/modernization.  A lot of the changes are just moving types to be simple data types.  One main type got heavily refactored and is called out.